### PR TITLE
Mount /var/data/ by default in Node Image Analyzer daemonset

### PR DESCRIPTION
**Pull request by luca-sd**
Some IBM Cloud OpenShift clusters use `/var/data` and `/var/data/crioruntimestorage` directory to store image data.
By adding it to the default NIA pod we can easily enable basic NIA functionality on those environments.

### CODE REVIEW

**Comment by nathan-b** [comment]
Would it make sense to mount this path as read-only? Or does the NIA need to write to this path as well?

**Comment by luca-sd** [comment]
Good point. `containers/storage` data needs to be mounted RW because access to some of its directories is governed by a lockfile. The new mountpoint is the target directory for those files in some deployments common in IBM cloud. Essentially, that's the same situation described a few lines above.

**Review by maxzerbini — APPROVED** [review]
LGTM

**Review by mattpag — APPROVED** [review]

File: agent_deploy/kubernetes/sysdig-image-analyzer-daemonset.yaml [hunk 1]
```diff
@@ -30,6 +30,10 @@ spec:
       - name: var-lib-containers-vol
         hostPath:
           path: /var/lib/containers
+      # Needed for some IBM OpenShift clusters which symlink /var/run/containers/storage to contents of /var/data by default
+      - name: vardata-vol
+        hostPath:
+          path: /var/data
```
Comment (fairoldi) [inline agent_deploy/kubernetes/sysdig-image-analyzer-daemonset.yaml:36]: what happens if this path does not exists? Has that been tested?

Comment (luca-sd) [reply agent_deploy/kubernetes/sysdig-image-analyzer-daemonset.yaml:36]: Nothing happens if the directory is missing. To validate this we have ran integration and smoke tests with this DS against regular Kops, OpenShift and GKE clusters via regression tests by @franco-nonne

**Review by mbreitung — APPROVED** [review]

**Review by fairoldi — APPROVED** [review]

**Comment by fairoldi** [comment]
Github is drunk, this was merged to master

**Pull request closed without merging**